PYME/Acquire/Hardware/AndorNeo/ZylaControlPanel.py

```diff
@@ -73,16 +84,20 @@ def __init__(self, parent, cam, scope):
         self.cam = cam
         self.scope = scope
         
-        self.ctrls = [EnumControl(self, cam.SimpleGainEnumInstance),
+        self.ctrls = [EnumControl(self, cam.SimplePreAmpGainControl, targetPropertyName='SimpleGainMode'),
                       EnumControl(self, cam.PixelReadoutRate),
                       BoolControl(self, cam.SpuriousNoiseFilter),
                       BoolControl(self, cam.StaticBlemishCorrection),
-                      EnumControl(self, cam.CycleMode),]
-        
+                      EnumControl(self, cam.CycleMode),
+                      EnumControl(self, cam.TemperatureStatus),]

```

Comment:
Temperature status would probably be better reported through the status bar (e.g. either by adding a callback to `microscope.StatusCallbacks`), or by re-working the exisiting camera temperature stuff in `microscope.genStatus()` to defer the status stuff a bit more to the camera / let it decide what to report for temperature etc ...

There might also be either performance or synchronisation issues in polling the temperature and/or temperature status through an Enum control. As it stands, the Zyla code has the temperature being queried only when`StartExposure()` is called, with the result cached and the cached value used for subsequent calls to `GetCCDTemp()`. It's been a while, but I'm pretty sure this was done this way on purpose to address a performance problem.

Reply:
Now addressed as a new method that can be added to `microscope.StatusCallbacks` in init file. The method is called `TemperatureStatusText` and I add below a screenshot that shows it in action. With the current SDK no performance issues apparently.

![pyme-zyla-sona-tweaks](https://user-images.githubusercontent.com/3750860/100919302-863ebd80-34d1-11eb-9232-34eab93aa3c8.png)